tests/settings_tests/tests.py

```diff
@@ -445,3 +447,25 @@ def test_tuple_settings(self):
             finally:
                 del sys.modules['fake_settings_module']
                 delattr(settings_module, setting)
+
+
+class TestDatabaseMirror(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_without_mirror(self):
+        Simple.objects.using('default').create()
+        Simple.objects.using('other').create()
+
+        self.assertEqual(Simple.objects.count(), 2)
+
+    def test_with_mirror(self):
+        databases = settings.DATABASES
+        databases['other']['TEST']['MIRROR'] = 'default'
+
+        with override_settings(DATABAES=databases):
+
+            Simple.objects.using('default').create()
+            Simple.objects.using('other').create()
+
+            self.assertEqual(Simple.objects.count(), 1)
```

Comment:
Is this right?

The two `create` calls create two objects in the same database. The count call fetches from that single database. You need a `2` here no?